### PR TITLE
Export Triples Laplace Denominator to Python Layer

### PR DESCRIPTION
**Pull request by andyj10224**
## Description
The `LaplaceDenominator` class is exported to the Python layer, but not `TLaplaceDenominator`. I have a Psi4Numpy version of a new (T) method I invented that won't work without this.

## Todos

- [x] Export `TLaplaceDenominator` to Python

## Questions
- [x] None

## Checklist
- [x] Tests added for any new features
- [x] [All or relevant fraction of full tests run](http://psicode.org/psi4manual/master/build_planning.html#how-to-run-a-subset-of-tests)

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

File: psi4/src/export_fock.cc [hunk 1]
```diff
@@ -102,6 +102,11 @@ void export_fock(py::module &m) {
         .def("denominator_occ", &LaplaceDenominator::denominator_occ, "docstring")
         .def("denominator_vir", &LaplaceDenominator::denominator_vir, "docstring");
 
+    py::class_<TLaplaceDenominator, std::shared_ptr<TLaplaceDenominator>>(m, "TLaplaceDenominator", "docstring")
+        .def(py::init<std::shared_ptr<Vector>, std::shared_ptr<Vector>, double>())
+        .def("denominator_occ", &TLaplaceDenominator::denominator_occ, "docstring")
+        .def("denominator_vir", &TLaplaceDenominator::denominator_vir, "docstring");
```
Comment (loriab) [inline psi4/src/export_fock.cc:108]: Fill in the docstring placeholders, please

Comment (andyj10224) [reply psi4/src/export_fock.cc:108]: There is nothing filled in for `LaplaceDenominator` either, should I fill it in for those also?

Comment (loriab) [reply psi4/src/export_fock.cc:108]: Sure, filling in any placeholders you have a good notion of always appreciated.

Comment (andyj10224) [reply psi4/src/export_fock.cc]: Comments added

**Review by loriab — APPROVED** [review]
lgtm, thanks!

**Review by JonathonMisiewicz — APPROVED** [review]

**Review by jturney — APPROVED** [review]

**Review by zachglick — APPROVED** [review]
LGTM

**Comment by JonathonMisiewicz** [comment]
Let us know if there's a reason why this isn't marked "ready to merge."

**Comment by andyj10224** [comment]
> Let us know if there's a reason why this isn't marked "ready to merge."

My bad, merge away!!!

**Pull request merged**